service/controller/app/v1/resource/chart/delete.go

```diff
@@ -40,37 +40,22 @@ func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange inte
 }
 
 func (r *Resource) NewDeletePatch(ctx context.Context, obj, currentState, desiredState interface{}) (*controller.Patch, error) {
-	delete, err := r.newDeleteChange(ctx, obj, currentState, desiredState)
+	del, err := r.newDeleteChange(ctx, obj, currentState, desiredState)
```

Comment:
`del` is nice pick? I need to learn more.

Reply:
I don't mind. We should just aim to have it consistent.